ebpf-for-windows-demo.sln

```diff
@@ -13,10 +13,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{9171E26A
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "cilium_test", "tests\cilium_tests\cilium_test.vcxproj", "{12161211-5463-4AA9-9B6E-552552B475DC}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ebpf-for-windows", "ebpf-for-windows", "{E69DC31A-4E39-4E5D-957F-FFDC31C69513}"

```

Comment:
Since we have removed the submodule and using nuget package, we can also delete this folder completely:

`https://github.com/microsoft/ebpf-for-windows-demo/tree/main/ebpf-for-windows`

This was added to compile ebpf-for-windows submodule.